pyiron_atomistics/vasp/base.py

```diff
@@ -386,12 +417,10 @@ def collect_output(self):
                 self.structure = self.get_final_structure_from_file(filename="CONTCAR")
             except IOError:
                 self.structure = self.get_final_structure_from_file(filename="POSCAR")
-            self._sorted_indices = np.array(range(len(self.structure)))
+            self._idx_pyiron_to_user = np.array(range(len(self.structure)))
```

Comment:
```suggestion
            self._idx_pyiron_to_user = np.arange(len(self.structure), dtype=int)
```

Should be equivalent and more direct.

But also, really? Maybe I'm just being daft here, but I don't understand how [0,...,n] can always be the pyiron-to-user map. This map should be the inverse map of user-to-pyiron and that can be pretty much arbitrary depending on species order, so how is the reverse map always this simple?

Edit: Also came back to this at the end of the review. I expanded the context and see that this is happening when we didn't have a structure originally and need to read it from file. So is this just some sort of special case that only gets hit when we're importing an externally executed vasp job or something? That's a condition under which I could see setting the map like this, since pyiron doesn't have any prior idea what other order the atoms could have been in, so what's read in is good enough.

Reply:
Yep - it's just for the case where we're importing external non-pyiron jobs into pyiron. In that case we just assign a naive map.

More specifically, there is no structure in the job, or if the structure has length 0 (empty list).